Product/Production/Common/CONNECTCommonWeb/src/main/java/gov/hhs/fha/nhinc/connectmgr/uddi/UDDITimerTask.java

```diff
@@ -67,7 +62,7 @@ public void run() {
         boolean bUDDIActive;
```

Comment:
Do we still need method anymore?  remove it if we don't.  Make sure we also remove uddi related in gateway.properties